pages/profile/permission/content/fields.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  permission: {
+    label: '',
+    options: {
+      basic: {
+        label: 'Basic',
+        hint: `* View details of all named people at the establishment.
+               * View and request amendments to the licences they hold.
+               * View details of projects they're working on.`
+      },
+      read: {
+        label: 'Intermediate',
+        hint: `* View details of all people and all licences at the establishment.
+               * View details of the establishment, including named people and Schedule of Premises.
+               * Request amendments to the licences they hold.`
+      },
+      admin: {
+        label: 'Admin',
+        hint: `* View and request amendments to all licences at the establishment.
+               * Manage user accounts and permissions at the establishment.`
+      }
+    }
+  }
+};
```

Comment:
This content already exists for inviting a user. We should only have one copy of it, and re-use accordingly.

Reply:
@lennym there are slight copy differences, I will ignore them

Reply:
👍 At this point I think it's fair to assume those are accidental and assume consistency. We can change later if we need to.